pallets/handles/src/lib.rs

```diff
@@ -765,10 +764,7 @@ pub mod pallet {
 					let (canonical_handle_str, _) =
 						Self::get_canonical_string_vec_from_base_handle(&base_handle_str);
 
-					return Self::validate_canonical_handle_contains_characters_in_supported_ranges(
-						&canonical_handle_str,
-					)
-					.is_ok();
+					return Self::validate_canonical_handle(&canonical_handle_str).is_ok();
```

Comment:
Not sure why this wasn't doing all the other canonical checks, but it should have been.